### PR TITLE
Feature/sample data

### PR DESCRIPTION
**Pull request by lauraguerra1**
Description

- Sample data was added for three recipes and three users. 
- Sample data was also added for the ingredients. 
- Every ingredient needed for each sample recipe is included.

### CODE REVIEW

**Comment by lauraguerra1** [comment]
[[Iteration1] Make a sample data set for testing](https://trello.com/c/ejMwlUkL/5-iteration1-make-a-sample-data-set-for-testing)

**Comment by lauraguerra1** [comment]
[[Iteration1] Make a sample data set for testing](https://trello.com/c/ejMwlUkL/5-iteration1-make-a-sample-data-set-for-testing)

File: src/data/sampleData.js [hunk 1]
```diff
@@ -0,0 +1,1109 @@
...
+        "ingredient": 2028,
+        "amount": 3
+      },
+      {
+        "ingredient": 4047,
+        "amount": 2
+      }
+    ]
+  }
+]
```
Comment (mbenfowler) [inline src/data/sampleData.js]: most files should have a newline at the bottom!  That's why this symbol is here

**Review by mbenfowler — CHANGES_REQUESTED** [review]

**Review by taranveersingh93 — APPROVED** [review]
This looks good.

File: src/data/sampleData.js (hunk 1 above)
Comment (lauraguerra1) [reply src/data/sampleData.js]: Line added!

**Review by mbenfowler — APPROVED** [review]

**Pull request merged**